From 7fa8cd027065eb1acc597fee79c1611f5adf4b10 Mon Sep 17 00:00:00 2001
From: Alex Naidis <alex.naidis@linux.com>
Date: Thu, 1 Dec 2016 20:15:51 +0100
Subject: [PATCH] binder: enable SDCLANG LTO

Enabling SDCLANG LTO for libbinder
since it is a critical part for
system performance.

Change-Id: Ib36d1c923c0b623e8f59aaf7e4287e9e5a40efbd
Signed-off-by: Alex Naidis <alex.naidis@linux.com>
---
 libs/binder/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libs/binder/Android.mk b/libs/binder/Android.mk
index 54797d9..d9d127b 100644
--- a/libs/binder/Android.mk
+++ b/libs/binder/Android.mk
@@ -61,6 +61,7 @@ LOCAL_MODULE := libbinder
 LOCAL_SHARED_LIBRARIES += liblog libcutils libutils
 
 LOCAL_CLANG := true
+LOCAL_SDCLANG_LTO := true
 LOCAL_SANITIZE := integer
 LOCAL_SRC_FILES := $(sources)
 ifneq ($(TARGET_USES_64_BIT_BINDER),true)
@@ -81,6 +82,7 @@ endif
 
 LOCAL_MODULE := libbinder
 LOCAL_STATIC_LIBRARIES += libutils
+LOCAL_SDCLANG_LTO := true
 LOCAL_SRC_FILES := $(sources)
 ifeq ($(BOARD_NEEDS_MEMORYHEAPPMEM),true)
 LOCAL_C_INCLUDES += \
